README.md

```diff
@@ -93,19 +93,33 @@ $ COSMOS_NODE=localhost yarn testnet
 ---
 
 ### Production
-Get the Gaia binary from [GitHub](`https://github.com/cosmos/gaia/releases`).
 
-Install [Docker](https://docs.docker.com/get-started/).
+Building requires that [Docker](https://www.docker.com/get-docker) is installed
+on your system.
 
-Build and run the app.
+Build the app:
+
+The first argument is a commit for Voyager (e.g., the tag `0.4.3`).  Optional
+arguments (such as `--platform`) must come afterwards.
+
+```bash
+yarn run build 0.4.3 --platform={darwin|win32|linux}
+```
+
+You can specify a commit hash for the version of Gaia to use if it has been prebuilt:
```

Comment:
Didn't we agree, that devs need to provide a build Gaia binary? Shouldn't the documentation reflect that? Maybe we can also add why we chose to do it that way.